code/modules/surgery/organs/tongue.dm

```diff
@@ -302,51 +319,95 @@
 
 /obj/item/organ/tongue/slime
 	name = "slimey tongue"
 	desc = "It's a piece of slime, shaped like a tongue."
 	say_mod = list("blorbles", "bubbles")
-	ask_mod = "inquisitively blorbles"
-	yell_mod = "shrilly blorbles"
-	exclaim_mod = "loudly blorbles"
+	ask_mod = list("inquisitively blorbles", "inquisitively blorbles")
+	yell_mod = list("shrilly blorbles", "shrilly blorbles")
+	exclaim_mod = list("loudly blorbles", "loudly bubbles")
 	toxic_food = NONE
 	disliked_food = NONE
 
 /obj/item/organ/tongue/slime/Initialize(mapload)
 	. = ..()
 	languages_possible = languages_possible_base += typecacheof(/datum/language/slime)
 
-/obj/item/organ/tongue/moth
-	name = "mothic tongue"
-	desc = "It's long and noodly."
-	say_mod = "flutters"
-	icon_state = "tonguemoth"
-	liked_food = VEGETABLES | DAIRY | CLOTH
-	disliked_food = FRUIT | GROSS
-	toxic_food = MEAT | RAW
-
 /obj/item/organ/tongue/teratoma
 	name = "malformed tongue"
 	desc = "It's a tongue that looks off... Must be from a creature that shouldn't exist."
 	say_mod = "mumbles"
+	ask_mod = "softly mumbles"
+	exclaim_mod = "sharply mumbles"
+	yell_mod = "loudly mumbles"
 	icon_state = "tonguefly"
 	liked_food = JUNKFOOD | FRIED | GROSS | RAW
 
 /obj/item/organ/tongue/podperson
 	name = "plant tongue"
 	desc = "It's an odd tongue, seemingly made of plant matter."
+	say_mod = "windily says" // like plants are blown by winds
+	ask_mod = "windily asks"
+	exclaim_mod = "windily exclaims"
+	yell_mod = "windily yells"
 	disliked_food = MEAT | DAIRY
 	liked_food = VEGETABLES | FRUIT | GRAIN //cannibals apparently
+
+// insect people
+/obj/item/organ/tongue/fly
+	name = "proboscis"
+	desc = "A freakish looking meat tube that apparently can take in liquids."
+	icon_state = "tonguefly"
+	say_mod = "buzzes"
+	ask_mod = "inquisitively buzzes"
+	exclaim_mod = "surprisingly buzzes"
+	yell_mod = "loudly buzzes"
+	taste_sensitivity = 25 // you eat vomit, this is a mercy
+	modifies_speech = TRUE
+	liked_food = GROSS | MEAT | RAW | FRUIT
+
+/obj/item/organ/tongue/fly/handle_speech(datum/source, list/speech_args)
+	var/static/regex/fly_buzz = new("z+", "g")
+	var/static/regex/fly_buZZ = new("Z+", "g")
+	var/message = speech_args[SPEECH_MESSAGE]
+	if(message[1] != "*")
+		message = fly_buzz.Replace(message, "zzz")
+		message = fly_buZZ.Replace(message, "ZZZ")
+	speech_args[SPEECH_MESSAGE] = message
+
+/obj/item/organ/tongue/bee
+	name = "proboscis"
+	desc = "A freakish looking meat tube that apparently can take in liquids, this one smells slighlty like flowers."
+	icon_state = "tonguefly"
+	say_mod = "buzzes" // nothing different from fly's
+	ask_mod = "inquisitively buzzes"
+	exclaim_mod = "surprisingly buzzes"
+	yell_mod = "loudly buzzes"
+	taste_sensitivity = 5
+	liked_food = VEGETABLES | FRUIT
+	disliked_food = GROSS | DAIRY
+	toxic_food = MEAT | RAW
+
+/obj/item/organ/tongue/moth
+	name = "mothic tongue"
+	desc = "It's long and noodly."
+	say_mod = "flutters"
+	ask_mod = "vaguely flutters"
```

Comment:
vaguely? doesn't really sound like they are asking a question

Reply:
since moffs don't speak something clearly, I intended their question was not clear too, but well it is just normal now, but I still want something like that...